src/pages/account.vue

```diff
@@ -3,7 +3,7 @@
     <oc-loader v-if="loading" />
     <div v-if="!loading" class="uk-width-3-4@m uk-container uk-padding">
       <div class="uk-flex uk-flex-between uk-flex-middle">
-        <span class="uk-text-lead" v-translate>Account</span>
+        <h1 class="uk-text-lead oc-page-title" v-translate>Account</h1>
```

Comment:
can we get rid of uk-text-lead here as well? The class oc-page-title shall hold all necessary styles for a page title.